testassets/InteropTestsWebsite/Program.cs

```diff
@@ -29,16 +31,29 @@ public static void Main(string[] args)
             CreateWebHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .ConfigureKestrel(options =>
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args)
+        {
+            var builder = WebHost.CreateDefaultBuilder(args);
+
+            // Support --port and --use_tls cmdline arguments normally supported
+            // by gRPC interop servers.
+            int port = int.Parse(builder.GetSetting("port") ?? "50052");
```

Comment:
not really proud of this, but it works. Is there a more idiomatic way? :-)

Reply:
[Command line configuration](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.2#command-line-configuration-provider)! This is added by default by the WebHost.CreateDefaultBuilder.

Reply:
Something like 
```c#
                .ConfigureKestrel((context, options) =>
                {
                    context.Configuration.GetValue<int>("port");
                    ...
                }
```

Reply:
Done. Thanks for help!